### PR TITLE
Include roles in userinfo, remove unused mapper

### PR DESCRIPTION
**Pull request by sumesh-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-idm/realm-exports/Client based auth.json [hunk 1]
```diff
@@ -43,9 +43,8 @@
             "realm-management":[
                "view-users",
                "query-users",
-               "manage-clients",
                "query-clients",
-               "manage-users"
```
Comment (sreehari-aot) [inline forms-flow-idm/realm-exports/Client based auth.json:48]: any reason to remove `manage-users`?   there is a `locale` endpoint that needs manage-users as client roles in the service account to update the locale user attribute. Please confirm @sumesh-aot

Comment (abhilash-aot) [reply forms-flow-idm/realm-exports/Client based auth.json:48]: Yes manage users may be needed for api call to change locale if the user .  @Bonymol-aot / @abilpraju-aot  cna you guys confirm on use wrt Internationalisation

Comment (abilpraju-aot) [reply forms-flow-idm/realm-exports/Client based auth.json:48]: manage-users is needed to update user locale related to internationalization @abhilash-aot

Comment (sumesh-aot) [reply forms-flow-idm/realm-exports/Client based auth.json:48]: Good catch, thanks, updated 👍

**Review by sreehari-aot — APPROVED** [review]
LGTM!

**Review by abhilash-aot — APPROVED** [review]
Lgtm

**Pull request merged**